public/js/Histogram.js

```diff
@@ -0,0 +1,95 @@
+var d3 = require('d3');
+var _ = require('lodash');
+_.mixin(require('lodash-deep'));
```

Comment:
Hahaha.. the package that lodash is forked from is called 'underscore'. Lodash has since surpassed underscore in pretty much every way, but underscore gets to keep the sensible name. :P